### ISSUE TITLE
Build commands not working

### ISSUE THREAD

**Issue by plambe8** [maximegris/angular-electron#329]
```ENOTDIR: not a directory, rmdir '/Users/plambe/Desktop/R2/angular-electron-etl/dist/mac/Electron.app/Contents/Frameworks/Electron Framework.framework/Libraries'
Error: ENOTDIR: not a directory, rmdir '/Users/plambe/Desktop/R2/angular-electron-etl/dist/mac/Electron.app/Contents/Frameworks/Electron Framework.framework/Libraries'
    at Object.rmdirSync (fs.js:700:3)
    at Observable.rxjs_1.Observable [as _subscribe] (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/@angular-devkit/core/node/host.js:188:28)
    at Observable._trySubscribe (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/Observable.js:30:22)
    at /Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/util/subscribeTo.js:22:31
    at Object.subscribeToResult (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/util/subscribeToResult.js:10:45)
    at MergeMapSubscriber._innerSub (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/operators/mergeMap.js:82:29)
    at MergeMapSubscriber._tryNext (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/operators/mergeMap.js:76:14)
    at MergeMapSubscriber._next (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/operators/mergeMap.js:59:18)
    at MergeMapSubscriber.Subscriber.next (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:67:18)
    at MapSubscriber._next (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/operators/map.js:55:26)
    at MapSubscriber.Subscriber.next (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:67:18)
    at Observable.rxjs_1.Observable.obs [as _subscribe] (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/@angular-devkit/core/node/host.js:263:21)
    at Observable._trySubscribe (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/Observable.js:30:22)
    at MapOperator.call (/Users/plambe/Desktop/R2/angular-electron-etl/node_modules/@angular/cli/node_modules/rxjs/internal/operators/map.js:32:23)```

I'm getting this error when trying to use any of the build commands. The dir definitely exists

**Comment by Leonardonline** [comment]
Try to delete `node_modules` folder and launch `npm install` again

**Comment by plambe8** [comment]
That seems to have given me more errors 
<img width="1357" alt="Screen Shot 2019-04-17 at 9 18 19 AM" src="https://user-images.githubusercontent.com/14362981/56290757-d3e69680-60f1-11e9-8ad7-e54fc6428443.png">
<img width="1408" alt="Screen Shot 2019-04-17 at 9 18 04 AM" src="https://user-images.githubusercontent.com/14362981/56290758-d3e69680-60f1-11e9-90c6-8479592a7de6.png">

From here, I ran npm install again to re-generate the node_modules but then I'm back to square one. For some reason npm start works but npm run electron:mac doesn't

**Comment by martin0** [comment]
There's issues with tar dependency

  High            Arbitrary File Overwrite
  Package         tar
  Patched in      >=4.4.2
  Dependency of   @angular-devkit/build-angular [dev]
  Path            @angular-devkit/build-angular > node-sass > node-gyp > tar
  More info       https://npmjs.com/advisories/803

Build issues because I don't have python 2
but npm start brings up electron with message app works!
